components/user-store/org.wso2.carbon.identity.user.store.configuration/src/main/java/org/wso2/carbon/identity/user/store/configuration/UserStoreConfigServiceImpl.java

```diff
@@ -396,4 +402,33 @@ private UserStoreDTO getUserStoreDTO(String domain, Boolean isDisable, String re
         userStoreDTO.setRepositoryClass(repositoryClass);
         return userStoreDTO;
     }
+
+    /**
+     * Checks whether the tenant is loaded if not loaded the tenant.
+     *
+     * Note: This is required only if tenant configurations
+     * need to be redeployed.
```

Comment:
Can we make this two lines into one line? WDYT?

Reply:
Felt that it was too long when its in two lines